src/modules/tests/test-data/test-data.selector.ts

```diff
@@ -96,13 +98,15 @@ export const sumManoeuvreFaults = (manoeuvres: Manoeuvres, faultType: Competency
   });
 };
 
-export const hasSeriousFault = (data: TestData, competency: Competencies) => data.seriousFaults[competency];
+export const hasSeriousFault = (
+  data: CatBUniqueTypes.TestData, competency: Competencies) => data.seriousFaults[competency];
 
-export const hasDangerousFault = (data: TestData, competency: Competencies) => data.dangerousFaults[competency];
+export const hasDangerousFault = (
+  data: CatBUniqueTypes.TestData, competency: Competencies) => data.dangerousFaults[competency];
 
-export const getTestRequirements = (data: TestData) => data.testRequirements;
+export const getTestRequirements = (data: CatBUniqueTypes.TestData) => data.testRequirements;
```

Comment:
should we not have a generic TestData as well for the common stuff. As faults, eta etc aren't category specific

Reply:
Changed it to use the Common TestData type